packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -56,6 +56,8 @@ export interface MenuItemProps extends VibeComponentProps {
   splitMenuItem?: boolean;
   "aria-label"?: AriaAttributes["aria-label"];
   submenuPosition?: SubmenuPosition;
+  tooltipId?: string;
+  shouldShowTooltip?: boolean;
```

Comment:
Do we need this prop? there is already logic in the component for this [here](https://github.com/mondaycom/vibe/blob/669b9483da1b81348469063dbae04c8634fd985c/packages/core/src/components/Menu/MenuItem/MenuItem.tsx#L101).